README.md

```diff
@@ -311,7 +312,7 @@ export class MyComponent {
 }
 ```
 
-## Dynamic Configuration
+### Dynamic Configuration
```

Comment:
It looks like this header was wrong, updated it to be the same level of heading as the other titles.